Resources/Prototypes/Entities/Objects/Weapons/Guns/Basic/wands.yml

```
@@ -111,6 +121,10 @@
     proto: BulletInstakillMagic
     capacity: 3
     count: 3
+  - type: Tag
+    tags:
+    - Deathwand
+    - WizardWand
```

Needs harmony comment
```suggestion
  # Harmony start
  - type: Tag
    tags:
    - Deathwand
    - WizardWand
  # Harmony end
```